src/main/resources/data/tfc/patchouli_books/field_guide/zh_cn.json

```diff
@@ -0,0 +1,652 @@
+{
```

Comment:
This shouldn't be committed here - this file is unused at runtime and only exists as a single location for localization data. My preference would be to move it to `/resources/lang/zh_cn.json` and make `generate_book.py` point to the `./lang` folder automatically.

Reply:
This is nice so that the translation files can be used for git without adding extra files